failure_detection.md

```diff
@@ -14,9 +14,9 @@ nav_order: 3
 {:toc}
 
 A Node entering an unready state after 5 minutes is an obvious sign that a
-failure occurred, but depending on your physical environment, workloads, and
-tolerance for risk, there may be other criteria or thresholds that are
-appropriate.
+failure occurred. Furthermore, there may be other criteria or thresholds that are
+appropriate rather than depending on your physical environment, workloads, and
```

Comment:
"rather than"? To my understanding this changess the meaning of this sentence a lot. If you want to make this a sentence on its own, I'd suggest to just remove the "but..." from the old version.

Reply:
It does change the meaning. I did it because I didn't fully understand the sentence, thus I tried to rephrase it.
There is no use of `rather than`? I thought the idea was to say that there are couple of criterias for identifying failure and we are not sure that the current ones are the right ones, thus `rather than` adds a comparison to the sentence.

Reply:
The meaning is: there may be other things to look at than just ready state. Those other things depend on your env.

With "rather than" I read there may be other things which do NOT depend on your env...

Reply:
What troubles me is that I miss a conjunction word between the sentences (a, and b)
a. depending on your physical environment, workloads, and tolerance for risk,
b. there may be other criteria or thresholds that are appropriate.

Reply:
I don't think there is a conjuction needed. It (the old version) has the same meaning as "there may be other criteria which depend on your env..."

Reply:
```suggestion
failure occurred. However there may be other criteria or thresholds that are more appropriate based on your particular physical environment, workloads, and tolerance for risk.
```

Does this read better?

Reply:
Yes 👍🏻